Gemfile.lock

```diff
@@ -377,4 +374,4 @@ RUBY VERSION
    ruby 2.3.0p0
 
 BUNDLED WITH
-   1.15.2
+   1.16.1
```

Comment:
@alexsoble Let's keep this matching Heroku - check out https://github.com/studentinsights/studentinsights/pull/1542 (most of those comments aren't relevant but the links to the Heroku docs explain why they fix this, and it'd probably be good to follow their advice unless we have reasons to do otherwise).

Reply:
Ooooh, good catch, thanks for pointing me to #1542. 👍